datafusion/substrait/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -292,10 +292,10 @@ async fn decimal_literal() -> Result<()> {
     roundtrip("SELECT * FROM data WHERE b > 2.5").await
 }
 
-#[tokio::test]
-async fn null_decimal_literal() -> Result<()> {
-    roundtrip("SELECT * FROM data WHERE b = NULL").await
-}
+// #[tokio::test]
+// async fn null_decimal_literal() -> Result<()> {
+//     roundtrip("SELECT * FROM data WHERE b = NULL").await
+// }
```

Comment:
this would work to avoid commenting this out:

```suggestion
#[tokio::test]
async fn null_decimal_literal() -> Result<()> {
    roundtrip("SELECT * FROM data WHERE b != NULL").await
}
```

or altenatively something like 
`roundtrip("SELECT CAST(null AS DECIMAL(3, 2)) FROM data").await` works as well

Reply:
Thanks @Blizzara . If we change it in this way, will it affect the purpose for which it was intended to test?